recipes/fundamentals/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "fundamentals" %}
+{% set version = "2.3.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4d2b539671687d2e27f94b6790fe0eb36ff08745c0b72f70fb752f42d5fd7ea
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - mysqlSucker=fundamentals.mysql.directory_script_runner:main
+    - yaml2db=fundamentals.mysql.yaml_to_database:main
+    - sqlite2mysql=fundamentals.mysql.sqlite2mysql:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - coloredlogs
+    - docopt
+    - future
+    - psutil
+    - python >=3.6
+    - python-dateutil
+    - pyyaml
+    - requests
+
+test:
+  imports:
+    - fundamentals
+    - fundamentals.commonutils
+    - fundamentals.download
+    - fundamentals.files
+    - fundamentals.mysql
+    - fundamentals.nose2_plugins
+    - fundamentals.renderer
+    - fundamentals.stats
+  commands:
+    - mysqlSucker --help
+    - yaml2db --help
+    - sqlite2mysql --help
+  requires:
+    - cov-core
+    - nose2
```

Comment:
Since you're not running the test suite, I don't think this is even necessary:
```suggestion
```

Reply:
Thank @dopplershift . Great user tag!